index.php

```diff
@@ -1,2 +1,82 @@
-<?php header("Location: template.php"); ?>
-<!-- This will need to be addOrder.php once the file is created -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="styles/styles.css" rel="stylesheet" type="text/css" />
+    <script type="text/javascript" src="scripts/addorder.js"></script>
+    <link rel="icon" href="assets/icons/SiOverleaf.svg" type="image/icon" />
+    <title>GoToGro | Add Order</title>
+</head>
+
+<body>
+    <?php
+    include_once("navbar.inc");
+    ?>
+    <div class="addorder">
+        <h1 class="white-text">Add New Order</h1>
+        <form method="post" action="processORDER.php" novalidate="novalidate">
+            <div id="row_1">
+                <div class="inputField">
+                    <label for="product_ID1">Product ID</label>
+                    <select name="product_ID1" id="product_ID1">
+                        <option value="placeholder" hidden>Please Select</option>
+                        <option value="Milk">Milk</option>
+                        <option value="KitKat">KitKat</option>
+                        <option value="Milo">Milo</option>
+                        <option value="Cheese">Cheese</option>
+                        <option value="Ice Cream">Ice Cream</option>
+                    </select>
```

Comment:
I may've missed a conversation, but were we going to have the ability to search in the drop down field too??

Reply:
We could, but it's a bit complicated, better keep it for later once the form is fully functioning (nice to have)